src/Workspaces/CSharp/Portable/Formatting/Rules/WrappingFormattingRule.cs

```diff
@@ -78,12 +78,6 @@ private void AddSpecificNodesSuppressOperations(List<SuppressOperation> list, Sy
             {
                 AddSuppressWrappingIfOnSingleLineOperation(list, tokens.Item1, tokens.Item2);
             }
-
-            var ifStatementNode = node as IfStatementSyntax;
-            if (ifStatementNode?.Else != null)
-            {
-                AddSuppressWrappingIfOnSingleLineOperation(list, ifStatementNode.Else.ElseKeyword, ifStatementNode.Else.Statement.GetFirstToken(includeZeroWidth: true));
```

Comment:
Should we have this if we have the end curly on the same line?  i.e. ```else { ... }```?  Or is this handled by something else?